typescript/nomad-deploy/config/testnets/shibuya.ts

```diff
@@ -0,0 +1,52 @@
+// Testnet Details: https://docs.astar.network/integration/network-details
+
+import * as dotenv from 'dotenv';
+
+import { ChainJson, toChain } from '../../src/chain';
+import { CoreConfig } from '../../src/core/CoreDeploy';
+import { BridgeConfig } from '../../src/bridge/BridgeDeploy';
+
+dotenv.config();
+
+const rpc = process.env.SHIBUYA_RPC;
+if (!rpc) {
+  throw new Error('Missing RPC URI');
+}
+
+const chainJson: ChainJson = {
+  name: 'shibuya',
+  rpc,
+  deployerKey: process.env.SHIBUYA_DEPLOYER_KEY,
+  domain: 7000,
+  gas: { price: '10000000000' },
+  chunk: 2000,
+  timelag: 5,
+};
+
+export const chain = toChain(chainJson);
+
+export const devConfig: CoreConfig = {
+  environment: 'dev',
+  updater: '0x4177372FD9581ceb2367e0Ce84adC5DAD9DF8D55',
+  watchers: ['0x20aC2FD664bA5406A7262967C34107e708dCb18E'],
+  recoveryManager: '0x24F6c874F56533d9a1422e85e5C7A806ED11c036',
```

Comment:
note to self: do we have the PK for this? (it's not the deployer key..)

Reply:
i am just going to change this to the deployer key too cuz tbh i don't know if any of us has the key to this https://github.com/nomad-xyz/nomad-monorepo/pull/58/commits/3f9288b13d487319e523e5216b241fc9f4bcbfa1